Sources/NIOHTTP2/HTTP2StreamMultiplexer.swift

```diff
@@ -205,6 +205,15 @@ public final class HTTP2StreamMultiplexer: ChannelInboundHandler, ChannelOutboun
         context.fireChannelWritabilityChanged()
     }
 
+    public func errorCaught(context: ChannelHandlerContext, error: Error) {
+        if let streamError = error as? NIOHTTP2Errors.StreamError,
+           let channel = self.streams[streamError.streamID] {
+            channel.receiveStreamError(streamError)
+        }
+
+        context.fireErrorCaught(error)
```

Comment:
Do we really want to unconditionaly forward it to the next channel handler even if we deliver it to one of our child channels?

Reply:
Yeah, I think we probably do: this ensures that errors don't suddenly go missing when they were reported before.